src/components/map/MapUser.js

```diff
@@ -1,12 +1,232 @@
 import React from 'react';
 import { withRouter } from 'react-router';
+import TabBar from '../../views/TabBar';
+import { Map as GoogleMap, GoogleApiWrapper, Marker } from 'google-maps-react';
+import Dog from '../../views/profile/Dog';
+import Button from '../../views/design/Button';
+import StatusIndicator from '../../views/design/StatusIndicator';
+
+const ALL_USERS = [
+  {
+    id: 1,
+    name: "User One",
+    bio: "I love all dogs, but Dalamtians are my absolute favorite! Totally open to watch your dogs while you are on holiday!",
+    status: "ONLINE",
+    distance: "15KM AWAY",
+    location: {
+      lat: 47.38507005180391,
+      lng: 7.944326876563231
+    },
+    dogs: [
+      {
+        id: 1,
+        name: "Bello",
+        sex: "MALE",
+        breed: "Dalmatian",
+        age: "6 Months",
+        imageUrl: "https://www.pdsa.org.uk/media/7888/dalmatian-gallery-outdoors-8-min.jpg"
+      },
+      {
+        id: 2,
+        name: "Winston",
+        sex: "MALE",
+        breed: "Dalmatian",
+        age: "3 Years",
+        imageUrl: "https://vetstreet-brightspot.s3.amazonaws.com/ee/140380a73111e0a0d50050568d634f/file/Dalmatian-2-645mk062311.jpg"
+      },
+      {
+        id: 3,
+        name: "Fifi",
+        sex: "FEMALE",
+        breed: "Dalmatian",
+        age: "4 Years",
+        imageUrl: "http://azure.wgp-cdn.co.uk/app-yourdog/posts/dalmatian.jpg"
+      },
+      {
+        id: 4,
+        name: "Jumper",
+        sex: "MALE",
+        breed: "Dalmatian",
+        age: "11 Months",
+        imageUrl: "https://dogtime.com/assets/uploads/gallery/dalmatian-dog-breed-pictures/10-water.jpg"
+      }
+    ]
+  },
+  {
+    id: 2,
+    name: "User Two",
+    bio: "I'm a user and I like dogs!!!",
+    status: "ONLINE",
+    distance: "1KM AWAY",
+    location: {
+      lat: 47.381019226154905, 
+      lng: 7.951315032221059
+    },
+    dogs: [
+      {
+        id: 3,
+        name: "Fifi",
+        sex: "FEMALE",
+        breed: "Dalmatian",
+        age: "4 Years",
+        imageUrl: "http://azure.wgp-cdn.co.uk/app-yourdog/posts/dalmatian.jpg"
+      }
+    ]
+  },
+  {
+    id: 3,
+    name: "User Three",
+    bio: "Lorem ipsum dolor sit amet",
+    status: "OFFLINE",
+    distance: "5KM AWAY",
+    location: {
+      lat: 47.38275533240448,
+      lng: 7.931205231766877
+    },
+    dogs: [
+      {
+        id: 2,
+        name: "Winston",
+        sex: "MALE",
+        breed: "Dalmatian",
+        age: "3 Years",
+        imageUrl: "https://vetstreet-brightspot.s3.amazonaws.com/ee/140380a73111e0a0d50050568d634f/file/Dalmatian-2-645mk062311.jpg"
+      }
+    ]
+  }
+];
+
+const FAKE_FETCH_USER = (id) => {
+  for (let i = 0; i < ALL_USERS.length; i++) {
+    if(ALL_USERS[i].id === id) {
+      return ALL_USERS[i];
+    }
+  }
+}
```

Comment:
Fakes retrieving the data from the backend, will be replaced once the client code for the API is ready.